modules/shapefile/src/dbf-loader.js

```diff
@@ -0,0 +1,30 @@
+import parseDBF from './lib/parse-dbf';
+/** @typedef {import('@loaders.gl/loader-utils').LoaderObject} LoaderObject */
+
+// __VERSION__ is injected by babel-plugin-version-inline
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+/** @type {LoaderObject} */
+export const DBFWorkerLoader = {
+  id: 'dbf',
+  name: 'DBF',
+  category: 'table',
+  version: VERSION,
+  extensions: ['dbf'],
+  mimeTypes: [],
+  options: {
+    shp: {
+      // Default to ASCII or UTF-8?
+      encoding: 'utf-8',
```

Comment:
`shapefile` defaults to `windows-1252`: https://github.com/mbostock/shapefile#shp2json_encoding

Reply:
Then let's set `windows-1252` as default. Turns out Windows1252 is just a 256 character encoding so decoding it is trivial see below (per https://stackoverflow.com/questions/42414839/convert-windows-1252-to-utf-8-with-js). 

You could also support UTF8 using TextDecoder and raw ASCII by omitting the string lookup below, but I don't see much point if we can't auto-detect. Or is the idea to have a UI with checkboxes to let the user select encoding before loading? Are there examples of non-windows-1252 files that we want to support?

```
const WINDOWS_1252 = '\u0000\u0001\u0002\u0003\u0004\u0005\u0006\u0007\b\t\n\u000b\f\r\u000e\u000f\u0010\u0011\u0012\u0013\u0014\u0015\u0016\u0017\u0018\u0019\u001a\u001b\u001c\u001d\u001e\u001f !"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~€�‚ƒ„…†‡ˆ‰Š‹Œ�Ž��‘’“”•–—˜™š›œ�žŸ ¡¢£¤¥¦§¨©ª«¬­®¯°±²³´µ¶·¸¹º»¼½¾¿ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖ×ØÙÚÛÜÝÞßàáâãäåæçèéêëìíîïðñòóôõö÷øùúûüýþÿ';

function fromWindows1252(binaryString) {
    var text = '';
    for (let i = 0; i < binaryString.length; i++) {
        text += WINDOWS_1252.charAt(binaryString.charCodeAt(i));
    }
    return text;
}
```

Reply:
OK, I saw the .cpg file comment, so ignore my questions above.

Reply:
The text encoder polyfill we have for node.js has a ton of encodings (not clear if it is the right ones) but it is enormous, we would probably want to load it (or at least the code pages) dynamically if were going to support a wide selection.

Reply:
We can't just pass `windows-1252` or `utf-8` to `new TextDecoder(encoding)`?

> Are there examples of non-windows-1252 files that we want to support?

Yes the `.cpg` file is there to describe the encoding. I assume that any recent shapefile would have utf-8 encoding, but earlier ones probably have windows-1252.

```
> cat PADUS2_0Fee_CO.cpg
UTF-8
```

Reply:
Yes that would be ideal. I may be wrong here, but I recall having problems in the past with TextDecoder only supporting UTF8 on some platforms - just double check at least on the 3 browsers we have Mac. It won’t work on IE and Edge of course but that is probably ok, the user can find their own polyfills

Reply:
Yes that would be ideal. I may be wrong here, but I recall having problems in the past with TextDecoder only supporting UTF8 on some platforms - just double check at least on the 3 browsers we have Mac. It won’t work on IE and Edge of course but that is probably ok, the user can find their own polyfills

Reply:
I can’t seem to find a list of encodings that would always be supported or even what their name strings would be assuming they are supported. Seems platform dependent? Need to check the actual standard.

Reply:
I can’t seem to find a list of encodings that would always be supported or even what their name strings would be assuming they are supported. Seems platform dependent? Need to check the actual standard.

Reply:
Again note in a comment in the code: what are the semantics / rationale here of specifying UTF8 given that latin1 is officially the default?

Reply:
I'm planning to switch this to `latin1` when I get the polyfills working

Reply:
Still think you should add the latin1 decoder in my comment just above here to make sure we always can handle that case (e.g. on IE11 and Edge).